### DISCUSSION TITLE
airflow db upgrade 2.5.1 to 2.6.2 fails: "psycopg2.errors.DuplicateTable: relation "idx_log_dttm" already exists"

### DISCUSSION THREAD

**Discussion by betabw** [apache/airflow#35299] [Q&A] [answered]
I am upgrading my airflow instance from 2.5.1 to 2.6.2. I have 2.6.2 installed, and I am trying to upgrade our postgres db. Airflow db upgrade fails with this message:

INFO  [alembic.runtime.migration] Running upgrade 290244fb8b83 -> 6abdffdd4815, add dttm index on log table
Traceback (most recent call last):
  File "/lib/python3.10/site-packages/sqlalchemy/engine/base.py", line 1905, in _execute_context
    self.dialect.do_execute(
  File "/lib/python3.10/site-packages/sqlalchemy/engine/default.py", line 736, in do_execute
    cursor.execute(statement, parameters)
**psycopg2.errors.DuplicateTable: relation "idx_log_dttm" already exists**

I can see in [docs](https://airflow.apache.org/docs/apache-airflow/stable/migrations-ref.html) that "add ddtm index on log table" was introduced at 2.6.0. 

But it appears that we already have that index at 2.5.1 (according to this error). I have tried downgrading and upgrading incrementally, and it always fails once we get to 2.6.0. Thank you in advance to anyone who is able to help!

**Comment by potiuk** [comment, marked as answer]
You must drop it manually I guess.

**Reply by betabw** [reply]
That worked. Thank you!
